webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubrics-manager.js

```diff
@@ -38,9 +38,12 @@ class SakaiRubricsManager extends RubricsElement {
       <div class="sak-banner-info"><sr-lang key="locked_message">locked_message</sr></div>
 
       <div class="row">
-        <div class="col-md-4 form-group">
+        <div class="col-md-5 form-group">
           <label for="rubrics-search-bar"><sr-lang key="search_rubrics">Search Rubrics by title or author</sr-lang></label>
           <input type="text" id="rubrics-search-bar" name="rubrics-search-bar" class="form-control" @keyup="${this.filterRubrics}">
+          <a id="rubrics-search-clear-button" tabindex="0" @click="${this.clearRubricsSearch}" href="javascript:void(0)" title="${tr("clear_rubrics_search")}">
+          <i class="fa fa-times-circle" aria-hidden="true"></i>
+      </a>
```

Comment:
this should be a button, right?

Reply:
@ottenhoff , I observe that the 'buttons' in the Rubrics actions grid ( weighted/non-weighted, toggle shared, copy, etc.) use anchor tags with role='button'. 

Would that suffice in the case of this feature request?  i.e. and edit like this to pull request: 

          <a id="rubrics-search-clear-button" role="button" tabindex="0" href="javascript:" @click="${this.clearRubricsSearch}" 
             class="linkStyle fa fa-times-circle" aria-hidden="false" title="Clear Rubrics Search" aria-label="Clear Rubrics Search">
          </a>

- Myles

Reply:
A further note - for my adding this feature to rubrics I borrowed the code from the Gradebook filter control (which uses an anchor tag) :            
               <a class="gb-student-filter-clear-button" wicket:message="title:filter.studentsclear" 
                                   href="javascript:void(0)">
                            <i class="fa fa-times-circle" aria-hidden="true"></i>
                </a>

Reply:
"From a high level, buttons are to be used for performing an in-page action and anchors/links are for navigating to a new page"

If you can't make it work with a button, that's fine, but when you see elements like this that are performing an on-page action, your first instinct should be *button*

Reply:
Thank you for the insight Sam. I have pushed an update to the branch / pull request adding in the role="button" which is consistent with the existing rubric action icons/"buttons".  (trying to make the new clear search element work using <button>, while semantically correct and proper, was running me into other problems to solve (styling, etc.) pressing against time constraints.)

Reply:
@ottenhoff Hi, my pull request failed the  'copy rubric' test of Sakai Server for Cypress from rubric.js test suite. Looking at the tests that did pass and the failing stack trace I am not understanding what might be the issue. 
Is there a way for me to re-run the test?

Reply:
@ottenhoff  - Sam, I have pushed up an inconsequential change (some whitespace) to force a retest and at this point it has completed and passed thy rubric.js test is failed earlier - so I guess that is some fluke. The rest of the Cypress test suite continues to run as of writing this.

Reply:
I have added:  role="button" so the new code snippet is as follows (and the integration tests have now all passed successfully)

          <a id="rubrics-search-clear-button" role="button" tabindex="0" href="javascript:" @click="${this.clearRubricsSearch}" 
             class="linkStyle fa fa-times-circle" title="Clear Rubrics Search" aria-label="Clear Rubrics Search">
          </a>

Reply:
@adrianfish how do you create a button with zero styling?